### PR TITLE
Feature/theory

### PR DESCRIPTION
**Pull request by hshahwan**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ import Methods from './Component/views/Methods';
 import KeyTerms from './Component/views/KeyTerms';
 import Workshop from './Component/views/Workshop';
 import Theory from './Component/views/Theory';
+import Persona from './Component/views/Persona';
```
Comment (teenie-quaggard) [inline client/src/App.js]: is this from another PR?

File: client/src/Component/views/About/index.js [hunk 2]
```diff
@@ -1,35 +1,31 @@
 import React from 'react';
 import Header from '../../Common/Header';
 import Footer from '../../Common/Footer';
-import { ContentAbout } from './index.style';
+import { Content } from './index.style';
```
Comment (teenie-quaggard) [inline client/src/Component/views/About/index.js:4]: Also saw this somewhere else

File: client/src/Component/views/YourScenario/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Header from '../../Common/Header';
+import Footer from '../../Common/Footer';
+import { Content } from '../About/index.style';
+import houses from '../../../assets/houses.svg';
```
Comment (teenie-quaggard) [inline client/src/Component/views/YourScenario/index.js:5]: also repeated from another Pr

**Review by teenie-quaggard — APPROVED** [review]
There are some overlaps with other PRs, but I'm happy for you to merge as long as you make sure that you keep the most updated versions of each file when you are fixing conflicts!

**Review by ali-7 — APPROVED** [review]

**Pull request merged**